packages/e2e-test-app/test/LegacyImageTest.test.ts

```diff
@@ -37,6 +37,17 @@ describe('LegacyImageTest', () => {
     const dump = await dumpVisualTree('image-container');
     expect(dump).toMatchSnapshot();
   });
+
+  test('AnimatedImageTestNotPlaying', async () => {
+    const dump = await dumpVisualTree('animated-image-container');
```

Comment:
Nit: consider only dumping the image, instead of the full container. That should make the snapshots a bit easier to follow and the tests resilient to things like button styling changes.

Reply:
I'm actually going to drop the test anyway. What I really wanted to test was whether the XAML `IsPlaying()` property or `AutoPlay` property would change, but unfortunately those don't seem to show up in the snapshots:
https://github.com/microsoft/react-native-windows/blob/main/packages/e2e-test-app/test/__snapshots__/LegacyImageTest.test.ts.snap#L38-L42

Reply:
fyi @rozele that you should be able to tell the snapshot test to ask for the IsPlaying property (or other properties) by using the `additionalProperties` param to `dumpVisualTree`, see e.g.: https://github.com/microsoft/react-native-windows/blob/main/packages/e2e-test-app/test/TextComponentTest.test.ts#L117